.gitignore

```diff
@@ -26,8 +26,12 @@ docs/_modules/*_module.rst
 
 devtools/docker-compose.site.yaml
 
+# intergration test files
 test/integration/inventory/hosts
 test/integration/inventory/group_vars/all.yaml
+bigiq-regkey-licenses.txt
+bigiq-regkey-licenses-unformated.txt
+bigiq-utility-licenses.txt
```

Comment:
i think we have secrets addon for github that should prevent this? Can you check if you got it installed? I think you can find it in out gitlab,  for now this can remain